src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -141,6 +143,7 @@ class FormPage extends React.Component {
       form,
       contentBeforeButtons,
       contentAfterButtons,
+      formConfig,
```

Comment:
Does `route.formConfig` work? I saw that referenced on line 28 on this page, but that may be different values. Would save a few more lines of not having to pass this in. Otherwise everything else looks great